apps/search-poc/package.json

```diff
@@ -43,40 +25,61 @@
   "eslintConfig": {
     "extends": "react-app"
   },
+  "jest": {
```

Comment:
Have you rebased? I don't think you should have this as a change as it's been merged in a previous PR already?

Reply:
Yeah, I've removed those changes.